ZitiUpdateService/UpdateService.cs

```diff
@@ -33,8 +33,6 @@ public partial class UpdateService : ServiceBase {
 
 		public bool useGithubCheck { get; private set; }
 
-		private static string[] expected_hashes = new string[] { "39636E9F5E80308DE370C914CE8112876ECF4E0C", "51D52749DA021F095B021DF8C09BD62C55C36F1F" };

```

Comment:
I was talking through this with andrew just now - we probably want to make some chages to this on the whole... TBD